converters/fromZigbee.js

```diff
@@ -4588,23 +4588,21 @@ const converters = {
                 if (msg.data.hasOwnProperty('currentPositionLiftPercentage') && msg.data['currentPositionLiftPercentage'] !== 50 ) {
                     // postion cast float to int
                     result.position = currentPosition | 0;
-                    result.position = options.invert_cover ? 100 - result.position : result.position;
                 } else {
                     if (deltaTimeSec < timeCoverSetMiddle || deltaTimeSec > timeCoverSetMiddle) {
                         // postion cast float to int
                         result.position = currentPosition | 0;
-                        result.position = options.invert_cover ? 100 - result.position : result.position;
                     } else {
                         entry.CurrentPosition = lastPreviousAction;
                         result.position = lastPreviousAction;
-                        result.position = options.invert_cover ? 100 - result.position : result.position;
```

Comment:
Why was this removed? This prevents users from inverting their covers.

Reply:
Indeed, I cleaned up the code a little too much to match my installation. In my case with Home Assistant, I had the right information in my "cover-card".
I will put the lines back, thank you for your vigilance.

Reply:
Can you re-add this?

Reply:
Yes I do It, with one line after the condition. https://github.com/Koenkk/zigbee-herdsman-converters/blob/92fd707c41465277b540a7969d7f7a77a380c7af/converters/fromZigbee.js view the line 4600.
If you prefer, I can put back this line in each condition ?

Reply:
Ah i see